XamlStyler.Console/XamlStyler.Console.csproj

```diff
@@ -27,6 +27,7 @@
     <IsWebBootstrapper>false</IsWebBootstrapper>
     <UseApplicationTrust>false</UseApplicationTrust>
     <BootstrapperEnabled>true</BootstrapperEnabled>
+    <TargetFrameworkProfile />
```

Comment:
Is this empty property required with the migration? Or left in by mistake?

Reply:
I think it's an addition when changing versions, but useless.